README.md

```diff
@@ -0,0 +1,14 @@
+# How to add DataBase in ASP.NET MVC Chart?
+
+This sample explains that how to add database in ASP.NET MVC Chart.
```

Comment:
Modified:

This sample explains that how to bind database to the ASP.NET MVC Chart control.